### PR TITLE
Fix follow/tail test [full ci]

### LINKED ISSUE #3301: Integration test failure: docker logs with follow and tail
```
==============================================================================
Test-Cases.Group1-Docker-Commands.1-08-Docker-Logs :: Test 1-08 - Docker Logs 
==============================================================================
Docker logs with follow and tail                                      | FAIL |
5 != 15
```

Seen in CI builds https://ci.vmware.run/vmware/vic/7196 and https://ci.vmware.run/vmware/vic/7214

**Logs:**

[Test-Cases.Group1-Docker-Commands.1-08-Docker-Logs-VCH-7196-4503-container-logs.zip](https://github.com/vmware/vic/files/609621/Test-Cases.Group1-Docker-Commands.1-08-Docker-Logs-VCH-7196-4503-container-logs.zip)

Can't seem to upload the entire log bundle or log.html since they're very large, so here's a screenshot of the failure from log.html:

failure

### PR DESCRIPTION
**Pull request by hmahmood**
Fixes #3301

### CODE REVIEW

**Comment by mdubya66** [comment]
please hold off on merging this unless we see the issue again on CI. 
also this requires an ok from @mhagen-vmware 
thanks

File: tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot [hunk 1]
```diff
@@ -61,11 +61,13 @@ Docker logs with follow and tail
     Should Be Equal As Integers  ${rc}  0
     # Wait for the first 10 lines to be logged
     Wait Until Keyword Succeeds  20x  200 milliseconds  Grep Logs And Count Lines  ${id}  5  10
+    # --tail=5 to skip the first 5 lines and --follow to wait for the rest
+    ${proc}=  Start Process  docker ${params} logs --tail 5 --follow ${id}  shell=True
```
Comment (hickeng) [inline tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot]: could still have a race here if the logs process starts tailing after the next step in the test is run.
Is there any way to check for non-nil output so we can validate this command has actually taken effect before the kill is dispatched?

Comment (hmahmood) [inline tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot]: There is this: http://robotframework.org/robotframework/latest/libraries/Process.html#Process%20Should%20Be%20Running

Comment (hmahmood) [inline tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot]: .... although I would assume that after we return from that line, that the process is running.

**Comment by mdubya66** [comment]
Now seen in builds 7217, 7218 and 7219. Approved for merge once we get the how figured out

**Comment by hickeng** [comment]
@hmahmood in the interests of CI stability can we have an issue open, issue check in the code, and test for >= 5 lines instead of == 15

**Comment by hickeng** [comment]
LGTM but I'm robot n00b

File: tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot [hunk 2]
```diff
@@ -63,11 +63,12 @@ Docker logs with follow and tail
...
     # kill -HUP will create another 5 lines of log output
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} kill -s HUP ${id}
+    Should Be Equal As Integers  ${rc}  0
     # --tail=5 to skip the first 5 lines and --follow to wait for the rest
-    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} logs --tail=5 --follow ${id}
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} logs --tail 5 --follow ${id}
     Should Be Equal As Integers  ${rc}  0
     ${linecount}=  Get Line Count  ${output}
-    Should Be Equal As Integers  ${linecount}  15
+    Should Be True  ${linecount} >= 5
```
Comment (anchal-agrawal) [inline tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.robot]: We should update the corresponding line in https://github.com/vmware/vic/blob/master/tests/test-cases/Group1-Docker-Commands/1-08-Docker-Logs.md as well.

**Comment by anchal-agrawal** [comment]
LGTM

**Comment by hmahmood** [comment]
:recycle:

**Pull request merged**